packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -501,6 +502,425 @@ void main() {
       expect(tester.widget<Checkbox>(checkbox).value, true);
     });
 
+  testWidgets('The checkbox on CheckboxListTile changes mouse cursor when hovered', (WidgetTester tester) async {
```

Comment:
I'm guessing this test is copied from `Checkbox`... The kind of tests needed here are "CheckboxListTile respects X", where X are each of the new params

Reply:
Thanks for the suggestion. Just updated the messages!